data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_abilities.lst

```diff
@@ -130,11 +130,15 @@ Devour Spell	KEY:Spelleater ~ Devour Spell			CATEGORY:Special Ability	TYPE:Spell
+Dragontouched			KEY:Dragonfire Adept ~ Dragontouched			CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 1st level, you gain Dragontouched (see page 18) as a bonus feat.		ABILITY:FEAT|AUTOMATIC|Dragontouched
+#Applying suggested fix for Breath Effect
+Breath Effect			KEY:Dragonfire Adept ~ Breath Effect			CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 2nd, 5th, 10th, 12th, 15th, and 20th level, you can select one of the breath effects in Table 2-2: Dragonfire Adept Breath Effects for which you meet the minimum level prerequisite. These breath effects can alter your breath weapon's damage type or area, or apply a condition to targets in place of damage. Each time you use your breath weapon, you can choose to apply any one breath effect that you know. The chosen effect either replaces the normal fire damage dealt by your breath weapon or replaces the standard area of your breath weapon. Some effects can be applied only to a cone-shaped breath weapon, and others only to a line-shaped breath weapon. You can't apply more than one breath effect to your breath weapon unless the effect specifically states otherwise. Also, you can't apply the same breath effect to your breath weapon in two consecutive rounds (though you still can use your normal fire breath weapon every round). See page 77 in Chapter 3: Draconic Magic for full descriptions of the breath effects.		BONUS:ABILITYPOOL|Dragonfire Adept Breath Effect|1+(DragonfireAdeptLVL/5)+(DragonfireAdeptLVL>=12)
+#Scales AC bonus does not follow a mathematical progression.  - applying with PREVAR instead
+Scales				KEY:Dragonfire Adept ~ Scales					CATEGORY:Special Ability	TYPE:Extraordinary.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 2nd level, your skin becomes thick and scaly, granting you a +2 bonus to your natural armor. The scales can be of any color or metallic hue; they are often (but not always) of a draconic hue that matches your outlook and alignment. This bonus improves to +3 at 8th level, to +4 at 13th level, and to +5 at 18th level. If you already have a natural armor bonus, use the higher of the two values.		DEFINE:ScalesArmorBonus|0	BONUS:COMBAT|AC|ScalesArmorBonus|TYPE=NaturalArmor		BONUS:VAR|ScalesArmorBonus|2		BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,8	BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,13	BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,18	
```

Comment:
Sorry - replace AC bonus with DR bonus above -
Also, it would increase their DR/Magic from sources other than Dragonfire
Adept, which is not the intended result


Jonathon Lee Walker

On Sun, Mar 20, 2016 at 11:47 AM, Jonny Walker <jonathon.l.walker@gmail.com>
wrote:

> The problem with that is that by Epic Level Handbook rules, Dragonfire
> Adept should not get AC bonuses at epic levels, and this would grant them
> and require whoever implemented the Epic Progression for this class to
> refactor it then anyway.
>
> Jonathon Lee Walker
>
> On Sun, Mar 20, 2016 at 12:12 AM, Bahamut <notifications@github.com>
> wrote:
>
>> In data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_abilities.lst
>> <https://github.com/BahamutDragon/pcgen/pull/445#discussion_r56759035>:
>>
>> > @@ -130,11 +130,15 @@ Devour Spell	KEY:Spelleater ~ Devour Spell			CATEGORY:Special Ability	TYPE:Spell
>> >  Weapon and Armor Proficiency	KEY:Dragonfire Adept ~ Weapon and Armor Proficiency	CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.25	DESC:Dragonfire adepts are proficient with simple weapons, but not with armor or shields. Like arcane spellcasters, a dragonfire adept wearing armor or using a shield incurs a chance of arcane spell failure (all invocations have somatic components).
>> >  Invocations				KEY:Dragonfire Adept ~ Invocations				CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.25	DESC:A dragonfire adept has a repertoire of attacks, defenses, and other abilities known as draconic invocations, which allow her to focus the draconic energy that suffuses her soul. A dragonfire adept can use any invocation she knows at will. A dragonfire adept's invocations are spell-like abilities; using an invocation is therefore a standard action that provokes attacks of opportunity. To avoid provoking such attacks, a dragonfire adept can use an invocation defensively by making a successful Concentration check. An invocation can be disrupted, just as a spell can be ruined during casting. If a dragonfire adept is hit by an attack while invoking, she is entitled to a Concentration check to successfully use the invocation, just as a spellcaster would be. Her invocations are subject to spell resistance unless an invocation's d
>>  escription specifically states otherwise. A dragonfire adept's caster level with her invocations is equal to her class level. She can dismiss any invocation as a standard action, just as a wizard can dismiss a spell. If an invocation allows a saving throw, its DC is 10 + the equivalent spell level + the dragonfire adept's Cha modifier. Since spell-like abilities are not spells, a dragonfire adept cannot benefit from the Spell Focus feat or from draconic feats that let her convert or spend an arcane spell slot to produce some other effect. She can, however, benefit from the Ability Focus feat (MM 303), as well as from feats that emulate metamagic effects for spell-like abilities. The four grades of draconic invocations, in order of their relative power, are least, lesser, greater, and dark. A 1st-level dragonfire adept begins with knowledge of one least invocation, gaining access to more invocations and higher grades as she attains levels. At any level when a dragonfire adept
>>   learns a new invocation, she can also replace an invocation she already knows with another invocation of the same or lower grade. See Draconic Invocations, below, for a list of available invocations. Unlike other spell-like abilities, draconic invocations are subject to arcane spell failure chance as described under Weapon and Armor Proficiency, above. Finally, just like warlocks (see Complete Arcane), dragonfire adepts can qualify for some prestige classes usually intended for spellcasters. For details, see the Dragonfire Adepts and Prestige Classes sidebar on page 24.	DEFINE:DragonfireAdeptInvocations|0	BONUS:VAR|DragonfireAdeptInvocations|1+(Dragonfire_AdeptLVL>=3)+(Dragonfire_AdeptLVL>=6)+(Dragonfire_AdeptLVL>=8)+(Dragonfire_AdeptLVL>=11)+(Dragonfire_AdeptLVL>=13)+(Dragonfire_AdeptLVL>=16)+(Dragonfire_AdeptLVL>=18)	BONUS:ABILITYPOOL|Dragonfire Adept Invocation|DragonfireAdeptInvocations
>> >  Breath Weapon			KEY:Dragonfire Adept ~ Breath Weapon			CATEGORY:Special Ability	TYPE:Supernatural.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.25	DESC:At 1st level, you gain a breath weapon that you can use at will as a standard action. Each time you use your breath weapon, you can choose whether it takes the form of a 15-foot cone or a 30-foot line. This breath weapon deals 1d6 points of fire damage; a successful Reflex save (DC 10 + 1/2 your class level + your Con modifier) halves the damage. As you gain levels, your breath weapon's damage increases, as shown on Table 2-1. You are immune to the effect of your own breath weapon (but not to other breath weapons that produce similar effects, even those of other dragonfire adepts). At 10th level, your cone-shaped breath weapon's range doubles to 30 feet, and your line-shaped breath weapon's range doubles to 60 feet.
>> > -Dragontouched			KEY:Dragonfire Adept ~ Dragontouched			CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 1st level, you gain Dragontouched (see page 18) as a bonus feat.
>> > -Breath Effect			KEY:Dragonfire Adept ~ Breath Effect			CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 2nd, 5th, 10th, 12th, 15th, and 20th level, you can select one of the breath effects in Table 2-2: Dragonfire Adept Breath Effects for which you meet the minimum level prerequisite. These breath effects can alter your breath weapon's damage type or area, or apply a condition to targets in place of damage. Each time you use your breath weapon, you can choose to apply any one breath effect that you know. The chosen effect either replaces the normal fire damage dealt by your breath weapon or replaces the standard area of your breath weapon. Some effects can be applied only to a cone-shaped breath weapon, and others only to a line-shaped breath weapon. You can't apply more than one breath effect to your breath weapon unless the effect specifically states otherwise. Also, you can't apply the same breath effect to your
>>  breath weapon in two consecutive rounds (though you still can use your normal fire breath weapon every round). See page 77 in Chapter 3: Draconic Magic for full descriptions of the breath effects.
>> > -Scales				KEY:Dragonfire Adept ~ Scales					CATEGORY:Special Ability	TYPE:Extraordinary.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 2nd level, your skin becomes thick and scaly, granting you a +2 bonus to your natural armor. The scales can be of any color or metallic hue; they are often (but not always) of a draconic hue that matches your outlook and alignment. This bonus improves to +3 at 8th level, to +4 at 13th level, and to +5 at 18th level. If you already have a natural armor bonus, use the higher of the two values.
>> > +Dragontouched			KEY:Dragonfire Adept ~ Dragontouched			CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 1st level, you gain Dragontouched (see page 18) as a bonus feat.		ABILITY:FEAT|AUTOMATIC|Dragontouched
>> > +#Applying suggested fix for Breath Effect
>> > +Breath Effect			KEY:Dragonfire Adept ~ Breath Effect			CATEGORY:Special Ability	TYPE:Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 2nd, 5th, 10th, 12th, 15th, and 20th level, you can select one of the breath effects in Table 2-2: Dragonfire Adept Breath Effects for which you meet the minimum level prerequisite. These breath effects can alter your breath weapon's damage type or area, or apply a condition to targets in place of damage. Each time you use your breath weapon, you can choose to apply any one breath effect that you know. The chosen effect either replaces the normal fire damage dealt by your breath weapon or replaces the standard area of your breath weapon. Some effects can be applied only to a cone-shaped breath weapon, and others only to a line-shaped breath weapon. You can't apply more than one breath effect to your breath weapon unless the effect specifically states otherwise. Also, you can't apply the same breath effect to your
>>  breath weapon in two consecutive rounds (though you still can use your normal fire breath weapon every round). See page 77 in Chapter 3: Draconic Magic for full descriptions of the breath effects.		BONUS:ABILITYPOOL|Dragonfire Adept Breath Effect|1+(DragonfireAdeptLVL/5)+(DragonfireAdeptLVL>=12)
>> > +#Scales AC bonus does not follow a mathematical progression.  - applying with PREVAR instead
>> > +Scales				KEY:Dragonfire Adept ~ Scales					CATEGORY:Special Ability	TYPE:Extraordinary.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 2nd level, your skin becomes thick and scaly, granting you a +2 bonus to your natural armor. The scales can be of any color or metallic hue; they are often (but not always) of a draconic hue that matches your outlook and alignment. This bonus improves to +3 at 8th level, to +4 at 13th level, and to +5 at 18th level. If you already have a natural armor bonus, use the higher of the two values.		DEFINE:ScalesArmorBonus|0	BONUS:COMBAT|AC|ScalesArmorBonus|TYPE=NaturalArmor		BONUS:VAR|ScalesArmorBonus|2		BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,8	BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,13	BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,18	
>>
>> Or you could do
>> BONUS:VAR|ScalesArmorBonus|2+((DragonfireAdeptLVL-3)/5)
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/BahamutDragon/pcgen/pull/445/files/bfe5cea1dc07bab8ab006bfbca670c1088ab7e16#r56759035>
>>
>
>